src/main/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPlugin.groovy

```diff
@@ -29,7 +29,7 @@ import org.gradle.api.file.FileCollection
  */
 class DockerRemoteApiPlugin implements Plugin<Project> {
     public static final String DOCKER_JAVA_CONFIGURATION_NAME = 'dockerJava'
-    public static final String DOCKER_JAVA_DEFAULT_VERSION = '3.0.6'
+    public static final String DOCKER_JAVA_DEFAULT_VERSION = '3.0.7'
```

Comment:
Yes, it is required. AFAIK, with `3.0.6` it is only possible to declare label via build-args. [Here](https://github.com/docker-java/docker-java/commit/305b73a9763f8d47bdc9698ecfaa9c844a6e087f#diff-d7f48445f5b749f480d3c3d0eb78f6ef) it has been enabled to declare the labels directly.